docs/manual/purity-checker.tex

```diff
@@ -10,7 +10,7 @@
 You do not need to run the Purity Checker directly.
 However you may run just the Purity Checker by
 supplying the following command-line options to javac:
-\code{-processor org.checkerframework.util.PurityChecker}.
+\code{-processor org.checkerframework.framework.util.PurityChecker}. 
```

Comment:
Please don't make unrelated whitespace changes.  This change adds a space at the end of the line.